playbooks/cluster_setup_debian.yaml

```diff
@@ -10,6 +10,7 @@
 - import_playbook: cluster_setup_libvirt.yaml
 - import_playbook: cluster_setup_ha.yaml
 - import_playbook: cluster_setup_keys.yaml
+- import_playbook: cluster_setup_auditd.yaml
```

Comment:
should be called in cluster_setup_hardened_debian because this is part of hardening.

Reply:
All tasks have been moved in debian hardening role.